dln/vi/model.py

```diff
@@ -692,7 +713,7 @@ def forward(self, x, y=None, infos=None, temperature=0.0, cost_only=False):
                 x_stripped = self.strip_options(x)
             else:
                 x_stripped = x
-
+            
```

Comment:
Remove empty space.

Reply:
That reminded me we should set up a linter/formatter.
https://github.com/microsoft/deep-language-networks/issues/21